js_modules/dagit/packages/app/src/NUX/CommunityNux.tsx

```diff
@@ -20,13 +21,20 @@ export const CommunityNux = () => {
     'communityNux',
     (data) => data,
   );
-  if (didDismissCommunityNux) {
+  const {data, loading} = useQuery(GET_SHOULD_SHOW_NUX_QUERY);
+  const [dismiss] = useMutation(SET_NUX_SEEN_MUTATION);
+
+  if (!isLocalhost()) {
+    return null;
+  }
```

Comment:
maybe worth a comment here since it might be the opposite of what people expect (only showing it on local dev)